src/types/types.ts

```diff
@@ -261,7 +261,7 @@ export interface CountryData {
 
 export const asMostRecentWallet = asObject({
   id: asString,
-  currencyCode: asString
+  tokenId: asEdgeTokenId
```

Comment:
We can't really change this without blowing away user settings - no good! We are kinda stuck with the way the data is.

Idea: Keep the data as-is, lookup the wallet by its walletId, and then search through enabled tokens to find a matching currency code. Then store the walletId + tokenId in memory. This keeps the data the same on disk, and even if the app has multiple tokens of the same name, it won't be a problem unless the user has them both *enabled*.

Once we have this migration code, we can write the data back to disk as walletId + tokenId, and the migration will be complete.